cpp/src/robotick/workloads/control/RemoteControlWorkload.cpp

```
@@ -65,6 +80,48 @@ namespace robotick
+		{
+			const float clamped_dead_zone = robotick::clamp(dead_zone, 0.0f, 0.99f);
+			if (robotick::abs(value) < clamped_dead_zone)
+			{
+				return 0.0f;
+			}
+
+			const float direction = value >= 0.0f ? 1.0f : -1.0f;
+			return (value - (direction * clamped_dead_zone)) / (1.0f - clamped_dead_zone);
+		}
+
+		static Vec2f apply_circle_to_square(const Vec2f& input)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)